CHANGELOG.md

```diff
@@ -9,8 +9,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 ### Fixed
+## [0.13.1]
+Fix case where cents are from 995 to 999 - 2017-06-07
```

Comment:
As 0.13.0 is not released yet, it will be all within 0.13.0.
But as I wrote in https://github.com/ravage84/SwissPaymentSlip/pull/27/commits/afea3e618d399fc52629e22a567081374170a2c4#r119148628 please add everythin under "Unreleased" and I'll update everything accodringly when I release a new version/tag.